.gitignore

```diff
@@ -1,13 +1,11 @@
+# Created by https://www.toptal.com/developers/gitignore/api/git,linux,macos,csharp,fsharp,windows,aspnetcore,dotnetcore,jetbrains+all,visualstudiocode
+# Edit at https://www.toptal.com/developers/gitignore?templates=git,linux,macos,csharp,fsharp,windows,aspnetcore,dotnetcore,jetbrains+all,visualstudiocode
```

Comment:
I usually use the latest `.gitignore` from the .NET SDK.
Does this `.gitignore` provide something that the .NET SDK's `.gitignore` is missing?